core/runner/src/test/java/com/cognifide/aet/runner/distribution/dispatch/CollectorJobSchedulerTest.java

```diff
@@ -96,15 +96,19 @@ public void add_expectMessageSent() throws Exception {
   }
 
 
-  @Ignore("Ignored for some unexpected test failure on Linux")
-  @Test(timeout = 2000l, expected = IllegalStateException.class)
-  public void add_whenAddedMessageExists_expectIllegalStateException() throws Exception {
+  @Test(expected = IllegalStateException.class)
```

Comment:
Thank you @toniedzwiedz ! `@Rule` approach applied. I will take a look at the suggested library soon.